docs/setup-guide/scripts/host_1.sh

```diff
@@ -0,0 +1,115 @@
+#! /bin/bash
```

Comment:
this will have a name conflict with scripts/host_1.sh added in the [the C2C PR](https://github.com/CentaurusInfra/fornax/pull/68)

Reply:
rename  it: from "host_1.sh" to "host_level_1.sh"